Bottom: 8d704774a072e672f9a490ce264b93cd7f47918f
Top:    e7aa1ad35528cfe87114fe7296e3220393bd6bc0
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 16:44:17 -0500

Enable TCP interface of docker


---
diff --git a/roles/docker-common/templates/daemon.json.j2 b/roles/docker-common/templates/daemon.json.j2
index 57de7390..6f6a7376 100644
--- a/roles/docker-common/templates/daemon.json.j2
+++ b/roles/docker-common/templates/daemon.json.j2
@@ -1,4 +1,5 @@
 {
   "log-driver": "json-file",
-  "log-opts": {"max-size": "10m", "max-file": "3"}
+  "log-opts": {"max-size": "10m", "max-file": "3"},
+  "hosts": ["unix:///var/run/docker.sock", "tcp://{{ ansible_default_ipv4["address"] }}:2375"]
 }
